### PR TITLE
feat: add NERPii

### PR DESCRIPTION
**Pull request by xuwq1993**

### CODE REVIEW

File: cognitive/src/main/scala/com/microsoft/ml/spark/cognitive/TextAnalytics.scala [hunk 1]
```diff
@@ -293,6 +293,18 @@ class NER(override val uid: String) extends TextAnalyticsBase(uid) with BasicLog
   def urlPath: String = "/text/analytics/v3.0/entities/recognition/general"
 }
 
+object NERPii extends ComplexParamsReadable[NERPii]
```
Comment (mhamilton723) [inline cognitive/src/main/scala/com/microsoft/ml/spark/cognitive/TextAnalytics.scala:296]: Can we rename to PII?

**Review by mhamilton723 — CHANGES_REQUESTED** [review]
Looks good one small nit!

File: cognitive/src/main/scala/com/microsoft/ml/spark/cognitive/TextAnalytics.scala (hunk 1 above)
Comment (xuwq1993) [reply cognitive/src/main/scala/com/microsoft/ml/spark/cognitive/TextAnalytics.scala]: done

**Review by mhamilton723 — APPROVED** [review]

**Pull request merged**